modules/Attendance/attendance_take_byCourseClassProcess.php

```diff
@@ -26,14 +26,14 @@
 //Module includes
 require_once __DIR__ . '/moduleFunctions.php' ;
 
-$gibbonCourseClassID=$_POST["gibbonCourseClassID"] ;
-$currentDate=$_POST["currentDate"] ;
+$gibbonCourseClassID=$_POST["gibbonCourseClassID"] ?? '';
+$currentDate=$_POST["currentDate"] ?? '';
 $today=date("Y-m-d");
 
-$moduleName = getModuleName($_POST["address"]);
+$moduleName = getModuleName($_POST["address"]) ?? '';
```

Comment:
Hi Harry, this is one spot where the null coalesce would need to be added to the variable itself rather than the function call, see my example below:

```suggestion
$moduleName = getModuleName($_POST["address"] ?? '');
```